pages/en/lb3/readmes/loopback-example-connector-soap.md

```diff
@@ -12,22 +12,25 @@ node stock-ws.js
 
 ### Example 2
 
-Get weather and forecast information for a given zip code:
+Get Periodic Table element information:
 
 ```
-node weather-rest.js
+node periodictable-rest.js
 ```
 
 ### Example 3
 
 To expose REST APIs to proxy the SOAP web services, run:
 
 ```
-node weather-ws.js
+node periodictable-ws.js
 ```
 
 then browse to [http://localhost:3000/explorer](http://localhost:3000/explorer)
 
+In API explorer's operations list, select `GET /PeriodictableServices/GetAtomicNumber` or `GET /PeriodictableServices/GetAtomicWeight`, then enter `elementName` and click on `Try it out`. 
```

Comment:
This README should not have been pulled into this PR.  Not sure how that happened, but I am in the process of completely rewriting the SOAP connector README with Rashmi.  IAC, as you probably know, we pull the READMEs from connectors by script into this repo, so we don't edit them here anyway.